### PR TITLE
[W5] [W13-4] Daniel Pyone Maung

### PR DESCRIPTION
**Pull request by sickerin**
Ready for Review

### CODE REVIEW

**Comment by chuaes** [comment]
Nice work! The coding style is consistent and follows the guideline.

**Comment by radhika-lakhtakia** [comment]
Looks great!

**Comment by wfxronald** [comment]
Great job in adding a new functionality to find persons whose phone numbers match a certain number sequence - this is especially useful when you want to find out whose phone number is it

File: test/java/seedu/addressbook/storage/StorageFileTest.java [hunk 1]
```diff
@@ -111,15 +111,15 @@ private StorageFile getTempStorage() throws Exception {
...
+                Collections.emptySet()));
         ab.addPerson(new Person(new Name("Betsy Crowe"),
-                                new Phone("1234567", true),
-                                new Email("betsycrowe@gmail.com", false),
-                                new Address("Newgate Prison", true),
-                                new HashSet<>(Arrays.asList(new Tag("friend"), new Tag("criminal")))));
+                new Phone("1234567", true),
+                new Email("betsycrowe@gmail.com", false),
+                new Address("Newgate Prison", true),
+                new HashSet<>(Arrays.asList(new Tag("friend"), new Tag("criminal")))));
```
Comment (ewaldhew) [inline test/java/seedu/addressbook/storage/StorageFileTest.java:122]: Avoid formatting unrelated parts of the code. Doing so can result in unrelated changes being included in the commit.

**Review by ewaldhew — COMMENTED** [review]
Tests and documentation are updated which is good, but the actual FnumCommand class seems to be missing?
Remember to stage before committing or else files won't be synced!
